WebApp/App_Start/Startup.Auth.cs

```diff
@@ -12,13 +12,16 @@
 using System.Threading.Tasks;
 using WebApp.Utils;
 using System.Web;
+using System.Net;
 
 namespace WebApp
 {
     public partial class Startup
     {
         public void ConfigureAuth(IAppBuilder app)
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3; // only allow TLSV1.2 and SSL3
```

Comment:
No! that's the wrong fix
We should never force the TLS, but leave it to the OS

What you need to do is update the runtime framework to more than 4.7.2 (that is to 4.8)